code/stable/glassbr/Website/GlassBR_SRS.html

```diff
@@ -2058,9 +2058,7 @@ <h3>Data Constraints</h3>
                     </tr>
                     <tr>
                       <td><em>a</em></td>
-                      <td>
-                        <em>a&thinsp;&gt;&thinsp;0 &and; a&thinsp;&ge;&thinsp;b</em>
```

Comment:
It appears to display on the next line, but I'm not quite sure if this is the "right" way to change things. I didn't actually touch much of the display/formatting code, so I haven't dug deep in this yet.

Reply:
This is fixed now